sympy/integrals/tests/test_integrals.py

```diff
@@ -1704,3 +1704,16 @@ def test_issue_4231():
 def test_issue_17841():
     f = diff(1/(x**2+x+I), x)
     assert integrate(f, x) == 1/(x**2 + x + I)
+
+def test_divergent():
```

Comment:
Could you please add checks for the following integrals. All should be divergent. 

```python
Integral(sin(x), (x, 0, oo))
Integral(1/x, (x, 0, 1))
Integral(1/x, (x, 1, oo))
Integral(1/x, (x, -1, 1))
```

I'm sure there are a lot more cases, but these kinds of things will probably be the most common.

Reply:
Ok sure i will add them